Core/Core/Features/Planner/CalendarMain/PlannerViewController.swift

```diff
@@ -136,10 +134,10 @@ public class PlannerViewController: UIViewController {
         offlineModeInteractor
             .observeIsOfflineMode()
             .sink { [weak self] isOffline in
-                guard let self else { return }
-                addButton.action = isOffline ? #selector(showOfflineAlert) : nil
-                addButton.menu = isOffline ? nil : addMenu
-                addButton.tintColor = isOffline ? .disabledGray : .textLightest.variantForLightMode
+                guard let self, isOffline else { return }
+                addButton.tintColor = .disabledGray
+                addButton.menu = nil
+                addButton.action = #selector(showOfflineAlert)
```

Comment:
When will the add button be enabled? If I go offline the button will be disabled but if I get back online it won't be enabled until I force restart the app.